sklearn/covariance/_robust_covariance.py

```diff
@@ -213,6 +213,35 @@ def _c_step(
     return location, covariance, det, support, dist
 
 
+def _consistency_factor(n_features, alpha):
+    """Multiplicative factor to make covariance estimate
+    consistent at the normal distribution, as described in [Croux1999]_.
+
+    Parameters
+    ----------
+    n_features : int
+        Number of features.
+
+    alpha : float
+        Parameter related to the support fraction.
+        This parameter must be in the range (0, 1).
+
+    Returns
+    -------
+    c_alpha : float
+        Scaling factor to make covariance matrix consistent.
+
+    References
+    ----------
+    .. [Croux1999] Influence Function and Efficiency of the Minimum
+        Covariance Determinant Scatter Matrix Estimator, 1999, Journal of
+        Multivariate Analysis, Volume 71, Issue 2, Pages 161-190
+    """
+    q_alpha = chi2.ppf(alpha, df=n_features)
+    c_alpha = gamma.cdf(q_alpha / 2, n_features / 2 + 1) / alpha
+    return 1.0 / c_alpha
```

Comment:
Could you detail a bit more how you arrived at this formula (eg equation number from Croux 1999) ?
Maybe this one (unfortunately unnumbered :)  ?
<img width="797" height="112" alt="Image" src="https://github.com/user-attachments/assets/ba06e12f-b2da-4993-9ab7-ddc5eb701138" />